packages/react/src/Column/Column.tsx

```diff
@@ -20,7 +20,7 @@ export type ColumnProps = {
 } & PropsWithChildren<HTMLAttributes<HTMLElement>>
 
 export const Column = forwardRef(
-  ({ as: Tag = 'div', children, className, gap = 'medium', ...restProps }: ColumnProps, ref: any) => (
+  ({ as: Tag = 'div', children, className, gap, ...restProps }: ColumnProps, ref: any) => (
     <Tag {...restProps} ref={ref} className={clsx('ams-column', `ams-column--gap-${gap}`, className)}>
```

Comment:
This renders `ams-column--gap-undefined` if you don't set `gap`

Reply:
Ah, yes. Added the condition.